Introductory Tests/EquationsOfMotion.py

```diff
@@ -55,8 +78,32 @@ def compute_motion(self, t: float, x: np.ndarray, risk_zone_radius: float = None
         Returns:
             (np.ndarray): K vector used for computing state at the following time step.
         """
-        a = self.compute_acceleration(x[0:3])
+        rotated_position = self.orbit_rotation(t, x[0:3])
+        a = self.compute_acceleration(rotated_position)
+
         kx = x[3:6]  
-        kv = a 
+        kv = a
+
         return np.concatenate((kx, kv))
+    
+
+    def orbit_rotation(self, t: float, x: np.ndarray) -> np.ndarray:
+        """ Rotates position x according to the analyzed body's real rotation.
+        Args:
+            t (float): Time value when position x occurs.
```

Comment:
in seconds I guess? would mention unit. Or one can also have it in the var name like `t_in_seconds` (or you can consistently just use SI units everywhere and trust that no one will ever mix it up :D )